homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -63,6 +66,10 @@
 ATTR_RC_ROTATION = 'rotation'
 ATTR_RC_VELOCITY = 'velocity'
 ATTR_STATUS = 'status'
+ATTR_ZONE_ARRAY = 'zone'
+ATTR_ZONE_REPEATER = 'reps'
```

Comment:
reps = repeats? Avoid shortening (and consolidate if another term than repeats is used in other platforms).

Reply:
Thanks for your tip!